content/news/last-information.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Last Information
+tags: [ 'nwerc 2023' ]
+date: 2023-11-22T15:00:00+01:00
+---
+
+The 2023 ICPC Northwestern Europe Regional Contest will take place this weekend, and we hope that you and your team are
+prepared for this great event.
+Please check for the latest [system information](/systems/) and the [schedule](/schedule).
+
+We want to remind you of the following:
+
+* The contest machines will be laptops, we do not provide you with an external keyboard. It is allowed to bring your own
+  keyboard (only one per team, only simple USB keyboards allowed). An external mouse will be provided, and it is not
+  allowed to replace it. Please hand in your keyboard at the registration desk and label the keyboard clearly with your
+  university, team name, and team number.
+* During the contest, you are only allowed to use your Team Reference Document (TRD) as reference material. You may
+  bring three identical copies of the document per team, clearly marked with the university and team name. For
+  specifications on the TRD, please check the [rules page](/rules/). Your TRD should be handed in at registration. It is
+  not
+  possible to print the TRD at the contest site.
+* This year there will be no water bottle in the goody bag. We advise you to bring your own refillable water bottle. The
+  water bottles can be filled at the taps in the toilets. The water from Dutch water taps can be drunk safely.
+* People using the sleeping accommodations on campus can report at the [X building](https://map.tudelftcampus.nl/poi/x/)
+  on Friday night at 23:00 and Saturday at 22:00. It is still possible to register for sleeping spots [here](https://forms.gle/c9qByf4SNYbWKop99).
```

Comment:
Is it a good idea to have this form posted online? :thinking:

Reply:
it is already on https://nwerc.eu/accommodations/